includes/gateways/class-wpsms-gateway-smses.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+namespace WP_SMS\Gateway;
+
+use WP_SMS\Gateway;
+use Exception;
+use WP_Error;
+
+class smses extends Gateway
+{
+    /**
+     * Base URL or web service endpoint for sending API requests.
+     *
+     * @var string
+     */
+    private $wsdl_link = "http://194.0.137.110:32161/bulk/sendsms";
+
+    /**
+     * The website or tariff page of the SMS service provider.
+     *
+     * @var string
+     */
+    public $tariff = "https://sms.es/";
+
+    /**
+     * Indicates whether the service provides a trial (test) mode.
+     * If true, you can test the service without purchasing credits.
+     *
+     * @var bool
+     */
+    public $unitrial = false;
+
+    /**
+     * The unit of SMS credit (e.g., "SMS" or "credit").
+     * This is usually set after receiving a response from the API.
+     *
+     * @var string|null
+     */
+    public $unit;
+
+    /**
+     * Indicates whether the service supports sending Flash SMS.
+     * Can be "enable" or "disable".
+     *
+     * @var string
+     */
+    public $flash = "enable";
+
+    /**
+     * Determines if the current message should be sent as a Flash SMS.
+     * Default is false, meaning messages are sent as normal SMS.
+     *
+     * @var bool
+     */
+    public $isflash = false;
+
+    /**
+     * Whether bulk SMS sending is supported
+     *
+     * @var bool
+     */
+    public $bulk_send = false;
+
+    /**
+     * Gateway fields
+     *
+     * @var array
+     */
+    public $gatewayFields = [
```

Comment:
Since these fields (username, password and from) are already available by default, we don’t need to customize them. For custom fields, we’ll define them properly in `$gatewayFields`